modules/calib3d/src/calibinit.cpp

```diff
@@ -479,8 +479,7 @@ bool findChessboardCorners(InputArray image_, Size pattern_size,
 
     bool found = false;
 
-    const int min_dilations = 0;
-    const int max_dilations = 7;
+    const bool is_plain = flags & CALIB_CB_PLAIN;
```

Comment:
There is windows build warning here:
```
opencv\modules\calib3d\src\calibinit.cpp(482): warning C4800: 'int': forcing value to bool 'true' or 'false' (performance warning) [build\modules\calib3d\opencv_calib3d.vcxproj]
```
https://pullrequest.opencv.org/buildbot/builders/precommit_windows64/builds/103677/steps/compile%20release/logs/warnings%20%281%29

Reply:
hi @mshabunin sorry but I do not have windows.
So I can't check on that platform.
the last push I did should resolve the warning as you suggested

Reply:
I meant casting the whole expression to bool: `static_cast<bool>(flags & CB_PLAIN)` (or C-style cast). Let's try the method from [VS documentation](https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-3-c4800?view=msvc-170) instead: `(flags & CALIB_CB_PLAIN) != 0`

Reply:
sorry I had not ... understood .. I put now(flags & CALIB_CB_PLAIN) != 0